### DISCUSSION TITLE
Failed to initiate the GitHub login process. Please try again. V 1.1.29.1869

### DISCUSSION THREAD

**Discussion by Moses-Code-Dev** [community/community#29320] [Copilot Conversations]
Any solution for:

Failed to initiate the GitHub login process. Please try again ( V 1.1.29.1869)

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

Please consult https://github.com/community/ community/discussions/29127

**Reply by Moses-Code-Dev** [reply]
I'm already logged in, the issue is that the plug-in is unable to load the local token or the sign in status.
